test/utils.test.ts

```diff
@@ -106,7 +107,8 @@ describe('constructPactFile', () => {
       pactConfig: {
         consumerName: 'ui-consumer',
         providerName: 'todo-api'
-      }
+      },
+      providerState: 'State'
```

Comment:
can we add new test cases to show the additional of state, if set, and defaulting to `''` if not set please.